ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { useCurrencyDisplay } from '../../../hooks/useCurrencyDisplay';
 import { EtherDenomination } from '../../../../shared/constants/common';
+import Box from '../box/box';
+import { Text } from '../../component-library';
```

Comment:
Import new TS version of `Box` component 
```suggestion
import { Text, Box } from '../../component-library';
```